README.md

```diff
@@ -101,24 +160,4 @@ open http://localhost:8080/test/
 In lieu of a formal styleguide, take care to maintain the existing coding style.
 
 ## Release History
-* Please see https://github.com/shama/stylus-loader/releases
-* 2.1.0 - Add support for stylus's include and set (@michaek)
-* 2.0.1 - Add peer dependency on stylus (@jchitel), fix PathCache for webpack 2 (@Unhelpful)
-* 2.0.0 - Remove dependency on stylus (@kossnocorps)
-* 1.6.1 - Remove version breaking change in 1.6.0
-* 1.6.0 - Remove dependency on stylus (@kossnocorps)
-* 1.3.0 - resolve use() calls (@mzgoddard), manual imports through path cache (@mzgoddard)
-* 1.2.0 - files in package.json (@SimenB), test running with testem (@mzgoddard), and some performance changes (@mzgoddard)
-* 1.1.0 - Pass through sourceMap option to stylus instead of defaulting to inline. Inherit source-map from devtool (@jordansexton).
-* 1.0.0 - Basic source map support (@skozin). Remove nib as dep. stylus is now a direct dep (as peerDependencies are deprecated).
-* 0.6.0 - Support loader prefixes when resolving paths (@kpdecker).
-* 0.5.0 - Disable Stylus parser caching in listImports (@DaQuirm). Update to stylus@0.49.2 and nib@1.0.4 as peerDependencies (@kompot).
-* 0.4.0 - Allow configuration of plugins through webpack config (@bobzoller). Update to stylus 0.47.2 (@shanewilson).
-* 0.3.1 - Fix when dependency (@tkellen)
-* 0.3.0 - Define url resolver() when "resolve url" option is true (@mzgoddard).
-* 0.2.0 - Now tracks dependencies for @import statements making cacheable work. Update stylus dep.
-* 0.1.0 - Initial release
-
-## License
-Copyright (c) 2014 Kyle Robinson Young  
-Licensed under the MIT license.
+* 2.1.0 - Initial release tracking `stylus-loader@2.1.0`, with fixed relative imports.
```

Comment:
Maybe add a note of about pre-fork releases with a link to https://github.com/shama/stylus-loader/releases